modules/core/src/main/java/org/apache/synapse/core/axis2/Axis2SynapseEnvironment.java

```diff
@@ -235,21 +235,29 @@ private void setSeverUnitTestMode(ServerContextInformation contextInformation) {
             //get unit test received port
             String requestPort = System.getenv("UNIT_TEST_PORT");
 
-            //set default port for TCP server connection
-            int unitTestingAgentPort = 9007;
-            if (requestPort != null && !requestPort.isEmpty()) {
-                try {
+            //check port for TCP server connection
+            int unitTestingAgentPort;
+            try {
+                if (requestPort != null && !requestPort.isEmpty()) {
                     unitTestingAgentPort = Integer.parseInt(requestPort);
-                } catch (NumberFormatException e) {
-                    log.error("Given port is not in valid format", e);
+
+                    if (unitTestingAgentPort > 1023) {
+                        UnitTestingExecutor testExecutor = UnitTestingExecutor.getExecuteInstance();
+                        testExecutor.setServerPort(unitTestingAgentPort);
+                        testExecutor.setSynapseConfiguration(this.synapseConfig);
+                        testExecutor.start();
+                    } else {
+                        log.error("Input TCP port \"" + unitTestingAgentPort + "\" must be higher than 1023, " +
```

Comment:
Number 1023 in this line and line 244 seems to be same. Therefore it is better to create a constant and use it in the two places.

Reply:
Done